collector/cpu_freebsd.go

```diff
@@ -74,50 +97,39 @@ func NewStatCollector() (Collector, error) {
 	}, nil
 }
 
-// Expose CPU stats using KVM.
+// Expose CPU stats using sysctl
```

Comment:
Style nit: periods after doc comments (also in the longer comment below after sentences).